app/code/Magento/Sales/Test/Unit/Model/Order/ItemTest.php

```diff
@@ -348,4 +347,25 @@ public function getItemQtyVariants()
             ]
         ];
     }
+
+    /**
+     * Test getPrice() method should returns float
```

Comment:
I think that you may remove the `should` here, as it actually returns a float value.
Please do the same for all the test methods.

Reply:
got it, fixed